packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -465,6 +466,10 @@ $Calendar-dayPicker-margin: 10px;
       @include text-align(right);
       top: 199px;
     }
+    //When date and month picker are side-by-side and "Go to today" button is visible no additional margin is needed
+    .wrap.goTodaySpacing{
+      margin-bottom: 0px;
```

Comment:
Could you add the spacing to the today button itself instead of adding it separately?